ui/src/app/certificate_requests/row.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from "react"
+const extractCSR = (csrPemString: string) => {
+    //TODO
+}
+
+const extractCert = (certPemString: string) => {
+    //TODO
+}
+
+type rowProps = {
+    id: number,
+    csr: string,
+    certificate: string
+}
+export default function Row({ id, csr, certificate }: rowProps) {
+    const [actionMenuOpen, setActionMenuOpen] = useState<boolean>(false)
+    const [detailsMenuOpen, setDetailsMenuOpen] = useState<boolean>(false)
+    return (
+        <tr>
+            <td className="" width={5} data-test-column="id">{id}</td>
+            <td className="">
+                <button 
+                    className="u-toggle p-contextual-menu__toggle is-small" 
```

Comment:
It is currently possible for multiple of them to be expanded, we probably want to limit it to 1

![image](https://github.com/canonical/gocert/assets/18486508/e6c9c3ce-f033-4681-b26a-b28916fd353b)

Reply:
done